src/components/editor/latex-textarea/MonacoContentManager.ts

```diff
@@ -93,33 +93,59 @@ export class MonacoContentManager {
       forceMoveMakers: true
     };
   }
-
-  public createDelta(
-    key: string, 
-    editorRef: MutableRefObject<editor.IStandaloneCodeEditor>, 
+  
+  public createDeltas(
+    selection: Selection,
+    insertedText: string,
+    isBackspace: boolean,
     generateCharacter: (val: string) => Character
-  ): Delta | undefined {
-    const { lineNumber: row, column } = editorRef.current.getPosition();
-    const currentOffset = this.positionToOffset({row, column});
-    const prevId = (currentOffset !== -1) ? this.undeletedDocument.at(currentOffset).id : null;
+  ) : Delta[] {
+    // Trim inserted text to prevent Ulysses pasting incident.
+    insertedText = (insertedText.length > 24) ? 
+      insertedText.substring(0, 24) : insertedText;
```

Comment:
Extract constant.

Also I think 24 is definitely too little, 128 maybe?

Reply:
moved to constant and set 128 for now